packages/calypso-e2e/src/lib/components/preview-component.ts

```diff
@@ -33,11 +34,31 @@ export class PreviewComponent {
 	}
 
 	/**
-	 * Close the theme preview pane.
+	 * Close the preview pane.
 	 *
 	 * @returns {Promise<void>} No return value.
 	 */
 	async closePreview(): Promise< void > {
 		await this.page.click( selectors.closeButton );
 	}
+
+	/**
+	 * Validates that the provided text can be found in the preview content. Throws if it isn't.
+	 *
+	 * @param {string} text Text to search for in preview content
+	 */
+	async validateTextInPreviewContent( text: string ): Promise< void > {
+		const frame = await this.getPreviewFrame();
+		await frame.waitForSelector( `text=${ text }` );
+	}
+
+	/**
+	 * Get the Iframe element handle for the preview content.
+	 *
+	 * @returns {Frame} Handle for the preview content Iframe
+	 */
+	private async getPreviewFrame(): Promise< Frame > {
+		const elementHandle = await this.page.waitForSelector( selectors.iframe );
+		return ( await elementHandle.contentFrame() ) as Frame;
```

Comment:
This is for the frame popover looking element, correct?

Reply:
It is! I added a clarification to the method header.